src/com/duckduckgo/mobile/android/activity/DuckDuckGo.java

```diff
@@ -762,6 +763,14 @@ private void stopAction() {
     	getSearchField().setCompoundDrawables(null, null, null, null);
 	}
 
+    private void showOnboardingInstruction() {
+        InstructionDialogFragment.newInstance(
```

Comment:
Perhaps pulling the instructionType int into a variable in the line above would be better for readability. Just an optional suggestion!